src/Api/Management/Controller/DeveloperController.php

```diff
@@ -104,7 +126,7 @@ protected function getBaseEndpointUri(): UriInterface
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
```

Comment:
IIRC it was intentionally `@inheritdoc` instead of the inlined version of it. Besides, it is an unrelated change, so I would propose changing this in a separate CR.

Reply:
https://github.com/apigee/apigee-client-php/issues/157

Reply:
Thanks, so please roll back unrelated changes from this PR if possible.

Reply:
@mxr576 if we roll back this change the `php-cs-fixer` throws `phpdoc_tag_type` error, so I think we should let this change go with this PR and for rest `phpdoc_tag_type` issue we can do it in #157

![image](https://user-images.githubusercontent.com/75600200/123755788-e49e1400-d8d9-11eb-8e39-cfa262667fac.png)